### PR TITLE
py sol for cf-1851F

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by freakin23** [comment]
AC: https://codeforces.com/contest/1851/submission/278352706

File: solutions/silver/cf-1851F.mdx [hunk 1]
```diff
@@ -79,5 +79,30 @@ int main() {
...
+
+```py
+t = int(input())
+for _ in range(t):
+	n, k = map(int, input().split())
+	arr = [*map(int, input().split())]
+	nums = [(arr[i], i + 1) for i in range(n)]
+	nums.sort()
+	mn, curr = 1e9, 0
+	for i in range(n - 1):
```
Comment (SansPapyrus683) [inline solutions/silver/cf-1851F.mdx:93]: can you make the names in line w/ c++

File: solutions/silver/cf-1851F.mdx [hunk 2]
```diff
@@ -79,5 +79,30 @@ int main() {
...
+		if nums[i][0] ^ nums[i + 1][0] < mn:
+			mn = nums[i][0] ^ nums[i + 1][0]
+			curr = i
+
+	print(
+		"{} {} {}".format(
```
Comment (SansPapyrus683) [inline solutions/silver/cf-1851F.mdx:99]: f-strings exist

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/silver/cf-1851F.mdx (hunk 1 above)
Comment (freakin23) [reply solutions/silver/cf-1851F.mdx]: done

File: solutions/silver/cf-1851F.mdx (hunk 2 above)
Comment (freakin23) [reply solutions/silver/cf-1851F.mdx]: ok

**Comment by freakin23** [comment]
AC: https://codeforces.com/contest/1851/submission/278411536

**Review by SansPapyrus683 — APPROVED** [review]

**Pull request merged**